blazor/datagrid/excel-like-filter.md

```diff
@@ -106,4 +106,54 @@ This **`FilterItemTemplate`** helps you to customize each CheckBox list element/
 The following screenshot represents Filter Item template
 ![Blazor DataGrid Filter with Item Template](./images/blazor-datagrid-filter-item-template.png)
 
+## Customize filter icon for filtered columns
+
+By default, DataGrid will display the in-built filtered icon with predefined styles after filtering the column. You can also customize the filtered icon using <b>.e-grid .e-filtered::before</b> class.
```

Comment:
By default, DataGrid will display the in-built filtered icon with predefined styles after filtering the column. The filtered icon can also be customized using <b>.e-grid .e-filtered::before</b> class.